workflow/rocoto_funcs/base.py

```diff
@@ -179,6 +181,7 @@ def wflow_task_part1(self,xmlFile): #write out part1 which excludes dependencies
     text=text+f'  <account>&ACCOUNT;</account>\n'
     text=text+f'  <queue>&QUEUE_DEFAULT;</queue>\n'
     text=text+f'  <partition>&PARTITION;</partition>\n'
+    text=text+f'  <native>-M &CLUSTER;</native>\n'
```

Comment:
Based on the previous comment, you can change `&CLUSTER` to `{cluster}`. Thanks!

Reply:
@guoqing-noaa I can't figure out how to pass in CLUSTER as c6 for the workflow tasks

Reply:
Sorry that I should have checked into this earlier. Please reverse the changes on this file.
Let's try to add it as a "native" directly and I will show you where to add it.

Reply:
Sorry that I should have checked into this earlier.
Let me check this and get back to you later today.